backend/tracim_backend/command/user.py

```diff
@@ -117,7 +117,7 @@ def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
         parser = super().get_parser(prog_name)
         parser.add_argument(
             "--send-email",
-            help="send mail to created user about user account creation (email parameter required an email notification need to be activated)",
+            help="send mail to created user(s) (you need to configure EMAIL-NOTIFICATION part in config file to use this feature)",
```

Comment:
If command not permit to create multiple user in same time, remove `(s)`.

Reply:
removed.